src/components/scenes/PluginViewScene.js

```diff
@@ -224,15 +238,49 @@ class PluginView extends React.Component<PluginProps, PluginState> {
     if (navState.loading) {
       return
     }
-    // TODO: improve handling of edge-ret URIs
-    if (navState.url.match(/edge-ret:\/\/plugins/)) {
+    const parsedUrl = parse(navState.url, {}, true)
+    if (parsedUrl.protocol === 'edge:' && parsedUrl.hostname === 'x-callback-url') {
+      switch (parsedUrl.pathname) {
+        case '/paymentUri':
+          this.props.abcWallet.parseUri(parsedUrl.query.uri)
+            .then(result => {
```

Comment:
I don't think we should be calling it "abcWallet" anymore, but `coreWallet`. ABC used to mean "Airbitz Core"

Reply:
fixed